src/website/src/pages/ec/components/button/demo/index.js

```diff
@@ -0,0 +1,13 @@
+import demoContentPrimary from '@ecl/ec-specs-button/demo/data--primary';
+import demoContentSecondary from '@ecl/ec-specs-button/demo/data--secondary';
+import demoContentCall from '@ecl/ec-specs-button/demo/data--call';
+import demoContentGhost from '@ecl/ec-specs-button/demo/data--ghost';
+import demoContentSearch from '@ecl/ec-specs-button/demo/data--search';
+import template from '@ecl/ec-twig-component-button/ecl-button.html.twig';
+import { correctSvgPath } from '@ecl/website-utils';
```

Comment:
I don't think this is used here

Reply:
It's used in this case
```js
export const buttonCall = template(correctSvgPath(demoContentCall));
```
Each time we have a spec demo data coming in with an .svg extension, this one will update the path accordingly